.github/workflows/finalize-release.yaml

```diff
@@ -13,48 +13,20 @@ on:
 
 jobs:
   finalize:
-    name: Cherry-pick Changelogs to master
+    name: Update master Changelogs
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: master
           fetch-depth: 0
       - name: Set Git Config
         run: |
           git config --local user.email 38288322+imodeljs-admin@users.noreply.github.com
           git config --local user.name imodeljs-admin
-      - name: Cherry-pick changelogs
-        run: |
-          git cherry-pick $commitid
-
-          # Clean up files that we don't want to cherry-pick changes back
-          for i in **/package.json; do git checkout --ours $i; done
-          git checkout --ours **/version-policies.json docs/changehistory/NextVersion.md
-
-          # Stage explicit files;
-          #   - Files resolved due to taking master's version
-          #   - All changelogs to remove duplicates from master
-          #     - They would be duplicated if the same PR is backported to the release branch
-          git add **/package.json **/version-policies.json common/changes docs/changehistory/NextVersion.md
-
-          commitMessage=$(git log --format=%B -n 1 $commitid)
-          git commit -m "$commitMessage Changelogs"
-
-          exitCode=$?
-          if [ $exitCode -ne 0 ]
-          then
-            echo Failed to cherry-pick changelogs for $commitMessage. Commit id is $commitid
-            exit $exitCode
-          fi
-
-          rush change --bulk --message "" --bump-type none
-          git add .
-          git commit --amend --no-edit
-
-          exit $?
+      - name: Run update-changelogs.sh
+        run: ./.github/workflows/automation-scripts/update-changelogs.sh
```

Comment:
i did this just to make it easier to run the git and awk cmds. I don't know how js runs shell cmds, but if that would be easier I could create a main function in the js script and run everything from there. Does that seem more appropriate?

Reply:
Was thinking about this as well. I understand the reason for the separation, but at the same time, one script is much more appealing. I'd try to combine them one way or another.

Reply:
after thinking about this some more I personally think it is better to use the shell cmds in the script to run the git cmds and file manipulations. So my secondary solution would be to inline the sh file into the github workflow, then call the js file inside the inlined shell script. This way we only create one js script, and no need for the extra .sh file. Does this sounds like a better solution?

Reply:
For me this comes down to does it make sense to write a bunch of shell commands in JS when the shell is right there for us to use. I don't think it does. (And JSON manipulation in the shell doesn't seem particularly appealing.) I'd rather have the extra file (not-inlined) and maybe throw both in a `changelogs/` folder inside of `automation-scripts`? 

I'm easily swayed on matters like this, though.

Reply:
> Please elaborate on why.

it is probably due to my inexperience with JS, and i am biased because when I need to do any work with file manipulation my first thought is always shell cmds with grep/awk/sed ect. In terms of efficiency and lines of code (like I said, this could be me not knowing a easy way to do this in JS) but finding all changelog files, copying them to a temp dir, changing the file names so they don't overwrite each other, then copying them back to the proper locations, seemed like it would require writing some fairly big functions, where as shell commands could do it in 3, and I also trust the performance of that much more then whatever I would try to do in JS.

Reply:
Initial tests look good. Even somehow managed to find some duplicates that must've slipped in back in 2.x i think. However, I would like to test it on a real patch locally before i'm confident in its functionality. However, should be good to review. Thanks Arun for that package, works wonders it seems